front_end/panels/js_profiler/js_profiler-meta.ts

```diff
@@ -18,14 +18,6 @@ const UIStrings = {
    *@description Command for showing the Profiler tool
    */
   showProfiler: 'Show Profiler',
-  /**
-   *@description Text for the performance of something
-   */
-  performance: 'Performance',
-  /**
-   *@description Command for showing the 'Performance' tool
-   */
-  showPerformance: 'Show Performance',
```

Comment:
Looks like this will just be a merge conflict when we upgrade Chrome. Why does the linter think these can be removed?

Reply:
They're unused so linter deletes them.

But just checked upstream and they have it there. I suppose Chrome team doesn't run these check commands then.

https://github.com/ChromeDevTools/devtools-frontend/blob/ac3e85ae479bf5c110a1261296ea5f515d9f2497/front_end/panels/js_profiler/js_profiler-meta.ts#L20-L27

I was thinking keeping linter happy would _help_ us avoid merge conflicts in the future, but it's doing the opposite.

Commands from https://chromium.googlesource.com/devtools/devtools-frontend/+/main/docs/README.md#useful-commands

(`check-loc` is also missing from [package.json](https://github.com/ChromeDevTools/devtools-frontend/blob/ac3e85ae479bf5c110a1261296ea5f515d9f2497/package.json#L39))

Will reach out to Chrome team and see if the above docs on formatting and lint checks are current.

Turning off linter in local editor for now.

Closing.